### ISSUE TITLE
"Could not be installed because it was not signed correctly"

### ISSUE THREAD

**Issue by Kamikato0** [Swirt/typertools#1]
Every time I try to install the extension I get the attached error. It says the extension is not signed correctly.

I tested it on another photoshop versions. It still not work.

I tried to install another extension to test and it worked but TyperTools didn't.

![resim_2021-04-14_011951](https://user-images.githubusercontent.com/82478177/114628436-83de4280-9cbf-11eb-815d-985250a3f917.png)

**Comment by Swirt** [comment]
Does the install.cmd script run correctly, without errors?
It switches Photoshop to developer mode, which doesn't require signing
extensions at all.

>

**Comment by Kamikato0** [comment]
No, script runs correctly. It doesn't have any error. I attached the screenshot.
![resim_2021-04-14_125942](https://user-images.githubusercontent.com/82478177/114692427-47433300-9d21-11eb-8647-d344bb05857c.png)
